### PR TITLE
Implement prettier for the project

### PR DESCRIPTION
**Pull request by kunall17**
This would help in code situations like in 

`const { narrow, subscriptions, unreadPrivateMessagesCount, onPressStreams, onPressPeople, } = this.props;`

and converting it to 
```
    const {
      narrow,
      subscriptions,
      unreadPrivateMessagesCount,
      onPressStreams,
      onPressPeople,
    } = this.props;
```

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@kunall17, it looks like you've already signed the Dropbox CLA. Thanks!

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "android-min": "react-native run-android",
     "android-max": "react-native run-android",
     "android-release": "react-native run-android --configuration 'Release'",
-    "test:lint": "eslint src/",
+    "test:lint": "prettier src/events/eventToAction.js",
```
Comment (borisyankov) [inline package.json]: Wuut?

**Comment by borisyankov** [comment]
👍 but see my comment.

File: package.json (hunk 1 above)
Comment (kunall17) [inline package.json]: Was just testing this, forget to remove this :)

**Comment by kunall17** [comment]
Done, btw for the fixing part should i do it in another PR or commit?

**Comment by borisyankov** [comment]
Yes, I am not even too sure we want to do it in a commit right now. Since this will break a lot of the history of who did work on what. Maybe for awhile people will run prettier on files they worked on, which not ideal, is closer to the ideal case.

**Comment by kunall17** [comment]
Hmm, true but this will now make the CI break, we must fix something for it.

**Pull request merged**